CityOfDetroit/bloom
# 1165/design items

## Issue #1165 

- Closes #1165
- Addresses #issue

## Description

1. Make max width and margins for language nav the same as the navbar
2. Footer: we should increase space between listing questions and general questions to 2rem
3. Footer-section component with logo and contact content should have max width of 1024px so text doesn't run too long
4. Menu icon not left aligned between md and lg screen widths
5.  Should have 1px line between the header and the content of the single listing page
6. Tooltip has line between bubble and arrow (see below)


## How Can This Be Tested/Reviewed?

Entry Point 1: http://localhost:3000/ | https://detroit-public-dev.netlify.app/
1. View header - language nav width and padding the same as navbar width and padding (Note: the border around the Sign Up button and lack of border around the right-most language button create the appearance that they are not aligned)
2. View footer - there should be a 2rem margin below listing questions section
3. Still on footer - left footer section should have a max-width of 1024px
4. Narrow screen width to between 768px and 1024px - the hamburger menu icon should remain right-aligned

Entry Point 2: http://localhost:3000/listings | https://detroit-public-dev.netlify.app/listings

5. Click into any listing - notice a 1px line between the Site Header and the content of the page (Note: I removed the top border of the process container to avoid clashing with the new divider line)
6. Still on listing page, click the tooltip - there should be no 1px border between the tooltip main rectangle and the tooltip arrow

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
